rbac/server/db/users_query.py

```diff
@@ -362,3 +362,15 @@ def fetch_username_match_count(conn, username):
         .run(conn)
     )
     return resource
+
+
+async def users_search_duplicate(conn, username):
+    """Search for packs based a string in the name field."""
```

Comment:
This doc string does not match what the function does

Reply:
addressed